taxi/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Car, Manufacturer
```

Comment:
add blank line between different import groups

use absolute import instead of relative